### ISSUE TITLE
Latest published gradle-module-plugin deploy task fails with exception

### ISSUE THREAD

**Issue by benmusson** [inductiveautomation/ignition-module-tools#47]
`0.1.2-SNAPSHOT` has a working `deployModl` task, but the currently published `0.1.1` fails with:
```
gradlew deployModl
...
FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':modules:example-perspective-component:deployModl' (type 'Deploy').
  - In plugin 'io.ia.sdk.modl' type 'io.ia.sdk.gradle.modl.task.Deploy' property 'module' has @Input annotation used on property of type 'RegularFileProperty'.
    
    Reason: A property of type 'RegularFileProperty' annotated with @Input cannot determine how to interpret the file.
    
    Possible solutions:
      1. Annotate with @InputFile for regular files.
      2. Annotate with @InputFiles for collections of files.
      3. If you want to track the path, return File.absolutePath as a String and keep @Input.
    
    For more information, please refer to https://docs.gradle.org/8.5/userguide/validation_problems.html#incorrect_use_of_input_annotation in the Gradle documentation.
```

This issue was resolved in https://github.com/inductiveautomation/ignition-module-tools/commit/163e0ee0485a0f0397b8a5bf10bad8a03be3b4d0, but publishing hasn't happened yet.

**Comment by brianeray** [comment]
Can you use `0.2.0-SNAPSHOT` from inductiveautomation/ignition-module-tools#43 for a little while?

It's published to `nexus.inductiveautomation.com/inductiveautomation-snapshots` so might require some configuration in your `settings.gradle[.kts]` to pull from there in addition to the default Gradle plugin portal.

And `0.2.0` should be published soon; that PR should get some final QA this week.

**Comment by benmusson** [comment]
Awesome, thanks. 
I'll probably just keep running a local build until 0.2.0 is out.

**Comment by brianeray** [comment]
That PR is merged now and `0.2.0` is published. See [this IA forum post for access](https://forum.inductiveautomation.com/t/how-can-we-perform-code-signing-using-an-hsm-with-module-signer/85640/21) as we have yet to establish a workflow to get it up to the Gradle Plugin Portal cleanly, under a company identity.

**Comment by brianeray** [comment]
I stand corrected. We have a GH workflow that is [still publishing to the Gradle Plugin Portal](https://plugins.gradle.org/plugin/io.ia.sdk.modl).
